addons/armory/ui/RscAttributes.hpp

```diff
@@ -62,12 +62,7 @@ class GVAR(dialog) {
             y = "SafeZoneY + (180 / 1080) * SafeZoneH";
             w = "(30 / 1920) * SafeZoneW";
             h = "(30 / 1080) * SafeZoneH";
-            onButtonClick = QUOTE(\
-                private _object = ((_this select 0) getVariable [ARR_2(QQGVAR(attachedObject),objNull)]);\
-                closeDialog 0;\
-                createDialog QQGVAR(newEquipment);\
-                ((findDisplay IDD_NEW_EQUIPMENT) displayCtrl IDC_BUTTON_CREATE) setVariable [ARR_2(QQGVAR(attachedObject), _object)];\
-            );
+            onButtonClick = QUOTE(private _object = ((_this select 0) getVariable [ARR_2(QQGVAR(attachedObject),objNull)]);closeDialog 0;createDialog QQGVAR(newEquipment);((findDisplay IDD_NEW_EQUIPMENT) displayCtrl IDC_BUTTON_CREATE) setVariable [ARR_2(QQGVAR(attachedObject), _object)];);
```

Comment:
This should be a function.

Reply:
moved to function openNewEquipmentDialog